libqtile/scripts/start.py

```diff
@@ -133,4 +134,11 @@ def add_subcommand(subparsers):
         dest='state',
         help='Pickled QtileState object (typically used only internally)',
     )
+    parser.add_argument(
+        '-b', '--backend',
+        default='x11',
+        dest='backend',
+        choices=libqtile.backend.CORES,
+        help=argparse.SUPPRESS,
```

Comment:
Why the SUPPRESS? Because we only have one core at the moment? I think it still have value to show the argument in the help, maybe it would encourage people to implement other backends. :)

Reply:
Yep that was my thinking, and that it might add a bit of noise, but, I guess you're right!